src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -123,31 +123,25 @@ public SmartMotorController getDeploy() {
     }
 
     public static class ClimberMap {
```

Comment:
It would be lovely if you could split this into a `frc.robot.maps.subsystems.ClimberMap` instead of having all the subsystem maps in one place